src/main/java/com/netgrif/application/engine/security/service/ISecurityContextService.java

```diff
@@ -6,7 +6,7 @@ public interface ISecurityContextService {
 
     void saveToken(String token);
 
-    void reloadSecurityContext(LoggedUser loggedUser);
+    void reloadSecurityContext(LoggedUser loggedUser, boolean forceRefresh);
```

Comment:
For better readability you should define two public methods that will hide the boolean param:
```
void reloadSecurityContext(LoggedUser loggedUser)
void forceReloadSecurityContext(LoggedUser loggedUser)
```
Then in the implementation you can have a private method with boolean param:
```
public void reloadSecurityContext(LoggedUser loggedUser) {
   reloadSecurityContext(loggedUser, false);
}
public void forceReloadSecurityContext(LoggedUser loggedUser) {
   reloadSecurityContext(loggedUser, true);
}
private void reloadSecurityContext(LoggedUser loggedUser, boolean forceRefresh) {
   ...
}
```

Reply:
Updated the code